platform/darwin/src/MGLFillExtrusionStyleLayer.h

```diff
@@ -0,0 +1,107 @@
+// This file is generated. 
+// Edit platform/darwin/scripts/generate-style-code.js, then run `make style-code-darwin`.
+
+#import "MGLStyleValue.h"
+#import "MGLVectorStyleLayer.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ Controls the translation reference point.
+ 
+ Values of this type are used in the `fillExtrusionTranslateAnchor` property of `MGLFillExtrusionStyleLayer`.
+ */
+typedef NS_ENUM(NSUInteger, MGLFillExtrusionTranslateAnchor) {
+    /**
+     The fill extrusion is translated relative to the map.
+     */
+    MGLFillExtrusionTranslateAnchorMap,
+    /**
+     The fill extrusion is translated relative to the viewport.
+     */
+    MGLFillExtrusionTranslateAnchorViewport,
+};
+
+/**
+ A fill-extrusion layer which allows customization of styling properties at runtime. You may 
```

Comment:
Would be nice to say “fill extrusion” instead of “fill-extrusion” here, but that can happen in a separate PR.